lib/view_model/repositories_view_model.dart

```diff
@@ -0,0 +1,24 @@
+import 'package:flutter/material.dart';
+
+import '../core/http/api_response.dart';
+import '../models/get_repositories/repository_rto.dart';
+import '../repository/get_repositories_repo.dart';
+
+class RepositoriesViewModel extends ChangeNotifier {
+  GetRepositoriesRepo repositoriesRepo = GetRepositoriesRepoImpl();
+  ApiResponse<List<RepositoryRTO>> repositories = ApiResponse.loading();
+
+  void _setRepositoriesState(ApiResponse<List<RepositoryRTO>> response) {
```

Comment:
Private and public order of methods seems off.

Reply:
done